src/components/Contributors.js

```diff
@@ -35,11 +36,11 @@ const Contributors = () => {
   const [contributorsList, setContributorsList] = useState([])
 
   useEffect(() => {
-    const list = data.contributors.map((item) => {
-      item.randomNumber = Math.floor(Math.random() * data.contributors.length)
-      return item
-    })
-    list.sort((a, b) => a.randomNumber - b.randomNumber)
+    const list = _.shuffle(
+      data.contributors.map((item) => {
```

Comment:
Why are we mapping these items? The map isn't altering them at all.

I think we can just do `const list = shuffle(data.contributors)`.

Again, let me know if I'm missing something.

Reply:
Agreed, and yeah, preferably with `import { shuffle } from "lodash"` throughout.